domain/camera/src/main/java/com/google/jetpackcamera/domain/camera/CameraXCameraUseCase.kt

```diff
@@ -123,6 +132,33 @@ constructor(
 
     private lateinit var imageCaptureUseCase: ImageCapture
 
+    /**
+     * Applies a CaptureCallback to the provided image capture builder
+     */
+    @OptIn(ExperimentalCamera2Interop::class)
+    private fun onCaptureCompletedCallback(imageCaptureBuilder: ImageCapture.Builder) {
+        val captureCallback = object : CameraCaptureSession.CaptureCallback() {
+            override fun onCaptureCompleted(
+                session: CameraCaptureSession,
+                request: CaptureRequest,
+                result: TotalCaptureResult
+            ) {
+                super.onCaptureCompleted(session, request, result)
+                try {
+                    if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.Q) {
+                        Trace.endAsyncSection(FIRST_FRAME_TRACE, FIRST_FRAME_COOKIE)
+                    }
+                } catch (_: Exception) {}
+            }
+        }
+
+        // Create an Extender to attach Camera2 options
+        val previewExtender = Camera2Interop.Extender(imageCaptureBuilder)
```

Comment:
Should this extender be on `Preview` rather than `ImageCapture`?

Reply:
i think this was a naming mistake